### PR TITLE
Adds Tips

### PR DESCRIPTION
**Pull request by JohnOxford**
Adds Tips

## Changelog (neccesary)
:cl:
add: Added Fallout related tips.
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -1,233 +1,28 @@
...
-As a Scientist, you can use the mutation toxin obtained from green slimes to turn yourself into a jelly mutant. Each subspecies has unique features - for example telepathic powers, duplicating bodies or integrating slime extracts!
-As a Scientist, you can maximize the number of uses you get out of a slime by feeding it slime steroid, created from purple slimes, while alive. You can then apply extract enhancer, created from cerulean slimes, on each extract.
-As a Scientist, you can disable anomalies by scanning them with an analyzer, then send a signal on the frequency it gives you with a remote signalling device. This will leave behind an anomaly core, which can be used to construct a Phazon mech!
-As a Scientist, researchable stock parts can seriously improve the efficiency and speed of machines around the station. In some cases, it can even unlock new functions.
-As a Scientist, you can generate research points by letting the tachyon-doppler array record increasingly large explosions.
-As a Scientist, getting drunk just enough will speed up research. Skol!
```
Comment (LetterN) [inline strings/tips.txt]: leave this, very important tip

File: strings/tips.txt [hunk 2]
```diff
@@ -1,233 +1,28 @@
...
-As a Revenant, your Overload Lights ability will only shock humans with lights if the lights are still on after a brief delay.
-As a Revenant, your Malfunction ability in general damages machinery and mechanical objects, possibly even emagging some objects. Experiment!
-As a Revenant, the illness inflicted on humans by Blight can be easily cured by lying down or with holy water, making it best used on targets that have no time to lie down, such as humans in combat.
-As a Swarmer, you can deconstruct more things than you think. Try deconstructing light switches, buttons, air alarms and more. Experiment!
-As a Swarmer, you can teleport fellow swarmers away if you think they are in danger.
-As a Morph, you can talk while disguised, but your words have a chance of being slurred, giving you away!
-As a Drone, you can ping other drones to alert them of areas in the station in need of repair.
-As a Ghost, you can see the inside of a container on the ground by clicking on it.
-As a Ghost, you can double click on just about anything to follow it. Or just warp around!
```
Comment (LetterN) [inline strings/tips.txt]: leave these ghosting tip

File: strings/tips.txt [hunk 3]
```diff
@@ -1,233 +1,28 @@
...
-When crafting most items, you can either manually combine parts or use the crafting menu.
-Suit storage units not only remove blood and dirt from clothing, but also radiation!
-Remote devices will work when used through cameras. For example: Bluespace RPEDs and door remotes.
+Crates can be booby-trapped by adding wire, shocking anyone who tries to open them.
+You can toggle the safety of your favorite shotgun by holding alt and clicking on it in your hotbar.
+Shift clicking on a person with a exposed face will show their faction affiliation at the bottom of the examine text.
+The Legate's armor has almost the same damage resistance as power armor. Be weary.
+On most advanced armor, the helmet has more damage resistance than the actual armor, sometimes it is useful to aim for the limbs.
+Armor piercing rounds will ignore power armor.
+Dynamite requires you to use a ignition tool on it to activate it. ( Torch / Zippo / Lighter / Welder )
```
Comment (JTGSZ) [inline strings/tips.txt]: Do we actually have dynamite?

**Comment by UndeadNappist** [comment]
Still need to make the changes requested before merging can happen.

**Pull request closed without merging**